infra/compile.py

```diff
@@ -111,8 +112,11 @@ def check_args(args, ctx):
     "int": "parseInt",
     "float": "parseFloat",
     "print": "console.log",
+    "AnimationClass": "AnimationClass"
 }
 
+SKIPPED_FNS = ["add_main_args"]
```

Comment:
Are we skipping it because it's hard to compile? If not, why not just compile it and then just don't call it?

Reply:
It's because it's work to make it compile. e.g. I'd have to mock the class that parses arguments.